gfx/pics.asm

```diff
@@ -1,18 +1,21 @@
 INCLUDE "includes.asm"
 
 
-; Unown pic pointers are assumed to start at the same address in a different bank.
-
 SECTION "Pic Pointers", ROMX
+
 PicPointers:: INCLUDE "gfx/pics/pic_pointers.asm"
 
+
 SECTION "Unown Pic Pointers", ROMX
+
+; Unown pic pointers are assumed to start at the same address as PicPointers,
+; but in a different bank.
```

Comment:
This should be enforced in the linkerscript as well.

Reply:
Done.